docs/plasma_calculator/index.rst

```diff
@@ -21,6 +25,6 @@ To invoke the app use ``plasma-calculator`` in the command line.
 By default this opens the app in a browser, bootstrapping
```

Comment:
I think you need to add the actual commands for launching the calculator.  I'm assuming you can do this from either the command line or a pythons session.  Should probably include the launch command for both these scenarios, but at a minimum show the CLI command now.  I'm guessing it looks something like...

```bash
python -m plasmapy plasma-calculator
```

Reply:
The command is `plasma-calculator`, which is stated just above this.  Would what's said in line 24 be enough?

Reply:
So, when `pip` installing `plasmapy`, `pip` creates the alias for the `plasma-calculator` command, correct?  I'm not going to request any changes at the moment, but I do think this section needs to be modified because its not obvious.  If I'm confused with the wording and how to use the functionality, then I'm confident many other users will be too.

---

Also, I get an error when trying to run the calculator on my windows machine...

![image](https://user-images.githubusercontent.com/29869348/176960605-65eb86ed-4d28-4236-938f-f545927b70f8.png)

My suspicion is whatever functionality is generating the file path is ignorant of a windows file separator, since the path and file name are mashed into one without a separator.